modules/qc/1.0/config/default.yaml

```diff
@@ -11,6 +11,7 @@ lcr-modules:
 
         options:
             samtools_stat: "" # can specify --remove-dups --remove-overlaps
+            samtools_coverage: "" # can specify a particular region of interest
```

Comment:
Can you give an example how you would specify this? I'm guessing it's just the usual samtools `-R chr:start-end`? And if you don't specify a region, does it give coverage of the whole genome?

Reply:
Thanks! Yes it is chr:start-end and  otherwise calculated for all contigs in bam header. I've added this to config for proper documentation!